testsuite/integration/web/src/test/java/org/jboss/as/test/integration/jsp/JspTagTestCase.java

```diff
@@ -44,12 +43,12 @@
 @RunAsClient
 public class JspTagTestCase {
 
-    private static final String RESULT = "This is a header\n" +
-            "\n" +
-            "\n" +
-            "<div>tag</div>\n" +
-            "\n" +
-            "Static content\n";
+    private static final String RESULT = "This is a header" + System.getProperty("line.separator") +
+    		System.getProperty("line.separator") +
```

Comment:
Although this might never happen in practice (given how the tests are run), but this change could still fail, if the test case runs on a Linux system and the WildFly server itself is running on a Windows system. The System.getProperty for `line.separator` would end up returning the client OS' line separator and not the server's. I think one way to solve this (in this specific testcase context) is to probably have a servlet exposed on some URL on the server side which returns the server's line separator character. Invoke this URL first in the  testcase and then use the returned value in the text that you want to compare for the other JSP responses.